react-rpg/src/Components/CardGrid/CardGrid.jsx

```diff
@@ -0,0 +1,26 @@
+import './cardgrid.css';
+import Card from '../Card/Card';
+
+
+const CardGrid = ({ selectedCharacter, items }) => {
+
+  if (selectedCharacter === null) {
```

Comment:
not bad but single returns are always kinda just better, you can also reuse the outer div and just make the innet
try to do 
```
return <div className="card-grid">
{  !selectedCharacter ? 
    Please select a character :
    <Card />
} 
</div>
```

Reply:
k